src/Symfony/Component/Serializer/Tests/Normalizer/GetSetMethodNormalizerTest.php

```diff
@@ -99,79 +99,11 @@ public function testDenormalizeWithObject()
         $this->assertEquals('bar', $obj->getBar());
     }
 
-    /**
-     * @group legacy
-     */
-    public function testLegacyDenormalizeOnCamelCaseFormat()
-    {
-        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
-
-        $this->normalizer->setCamelizedAttributes(array('camel_case'));
-        $obj = $this->normalizer->denormalize(
-            array('camel_case' => 'camelCase'),
-            __NAMESPACE__.'\GetSetDummy'
-        );
-
-        $this->assertEquals('camelCase', $obj->getCamelCase());
-    }
-
     public function testDenormalizeNull()
     {
         $this->assertEquals(new GetSetDummy(), $this->normalizer->denormalize(null, __NAMESPACE__.'\GetSetDummy'));
     }
 
-    /**
-     * @group legacy
-     */
-    public function testLegacyCamelizedAttributesNormalize()
```

Comment:
If you just remove the test, we loose a test proving that NameConverters are supported and really used by the normalizer.

IMO this test should be converted to use directly the NameConverter instead of `setCamelizedAttributes()` but not totally removed.
Same for the other normalizer.